plugin.video.videodevil/resources/catcher.list

```diff
@@ -102,17 +102,25 @@ build=%s|Referer=https://www.youporn.com/
 quality=standard
 ########################################################
 title=extremetube.com
-target=quality_480p":"(http[^"]+)
+target=quality_(?:240|180)p":"([^"]+)
 url=%s
 header=Cookie|age_verified=1
+action=prefix
+quality=low
+target=quality_(?:480|360)p":"([^"]+)
+url=%s
+header=Cookie|age_verified=1
+action=prefix
 quality=standard
-target=quality_720p":"(http[^"]+)
+target=quality_1080p":"([^"]+)
 url=%s
 header=Cookie|age_verified=1
+action=prefix
 quality=high
-target=quality_1080p":"(http[^"]+)
+target=quality_720p":"([^"]+)

```

Comment:
swap the 1080 and the 720p entry, so that the 1080p is at the bottom. That way if there are videos with both resolutions and a low resolution the list is sorted by resolution. You get a list like 
low
high
high

Just like it was before ...

Reply:
Done